swap/src/cli.rs

```diff
@@ -47,6 +47,9 @@ pub enum Command {
         receive_bitcoin: bitcoin::Amount,
     },
     BuyXmr {
+        #[structopt(short = "p", long = "connect-peer-id")]
```

Comment:
I don't think it makes to have  peer-id in the CLI if we don't have a way to configure multiple multiaddr's for a peer.

Reply:
Multiple multiaddr for a CLI sounds like a feature.

Reply:
What I mean is that it looks like that can be a future feature.